ui/src/main/webapp/src/app/project/migration-project-form.component.ts

```diff
@@ -49,6 +50,10 @@ export class MigrationProjectFormComponent extends FormComponent implements OnIn
             }
 
             this.isInWizard = flatRouteData.data.hasOwnProperty('wizard') && flatRouteData.data['wizard'];
+            this.isCreating = flatRouteData.data.hasOwnProperty('creating') && flatRouteData.data['creating'];
+
+            if(this.isCreating)
+                this._migrationProjectService.deleteProvisionalProjects().subscribe(res => {});
```

Comment:
To add to it all, I think projects validation should probably not be done through ajax, but rather by loading all the projects to the page and iterating that in JS. Sending requests for each key pressed is ridiculous. And I haven't seen a way to reduce it through ReactJS as the Angular async validator works with Promise. But that's probably something to deal with later.